qiskit/ignis/verification/randomized_benchmarking/rb_utils.py

```diff
@@ -43,6 +47,7 @@ def count_gates(qobj, basis, qubits):
     Additional Information:
         nQ gates are counted in each qubit's set of gates.
     """
+    warn('The function `count_gates` will be deprecated.', DeprecationWarning)
```

Comment:
The deprecation warning method should say what to replace its functionality with (if there is equivalent functionality).

Reply:
This function is called only by gate_per_clifford function. For readability of the code, I integrated this into gate_per_clifford function and deprecated this. I'll add this context to the warning.